dir2cast.php

```diff
@@ -1401,11 +1501,12 @@ public static function display($message, $errfile, $errline)
             }
             else
             {
-                header("Content-type: text/plain"); // reset the content-type
+                // define('CLI_ONLY') || !ini_get('html_errors')
```

Comment:
To be removed?

Reply:
This is actually just a description of when this branch of the if/else is triggered. I will clarify that in the comments. Thanks for the feedback!

Reply:
Fixed in 161c3d0